src/extension.ts

```diff
@@ -435,7 +435,7 @@ export class Ext extends Ecs.System<ExtEvent> {
         this.column_size = this.settings.column_size() * this.dpi;
         this.row_size = this.settings.row_size() * this.dpi;
 
-        if (this.settings.active_hint() && !this.active_hint) {
+        if (this.settings.active_hint() && !this.active_hint && utils.is_wayland()) {
```

Comment:
Should this be `!utils.is_wayland()`?

(I'm not really familiar with exactly what this code is supposed to be doing, but I assume it shouldn't be changing behavior on X).

Reply:
You're right. This feature should be disabled for Wayland, for now.